Show coloured tree-diff output in ImpTests (#4497)
input-output-hk/cardano-ledger: eras/shelley/impl/CHANGELOG.md

### `testlib`

* Remove `minMajorPV` and `maxMajorPV` from `Constants`

> * Change `logEntry` to take a `Doc AnsiStyle` instead of a `String`

Reviewer: ```suggestion
* Add `logDocEntry` that takes a `Doc AnsiStyle` instead of a `String`
```

### `testlib`

* Remove `minMajorPV` and `maxMajorPV` from `Constants`

> * Add `logDocEntry` that takes a `Doc AnsiStyle` instead of a `String`

Reviewer: ```suggestion
* Add `logDoc` that takes a `Doc AnsiStyle` instead of a `String`
* Rename `logEntry` to `logString`
```
